track/gbf.py

```diff
@@ -74,27 +85,52 @@ def _place_order_with_GBF(order_json):
     """
     # make post request to GBF
     # By default requests should be made as "test" via an environment variable.
-    # Once we go live, the environemnt variable needs to be set to true explictly,
+    # Once we go live, the environemnt variable needs to be set to true explictly.
+    order_number = json.loads(order_json)['orders'][0]['orderNumber']
```

Comment:
this seems way more brittle than just passing the order number as parameter into the function